packages/scandipwa/src/util/Product/Extract.js

```diff
@@ -161,6 +161,20 @@ export const getProductInStock = (product, parentProduct = {}) => {
     return stockStatus !== STOCK_TYPE.OUT_OF_STOCK && (inStock || stockStatus === STOCK_TYPE.IN_STOCK);
 };
 
+/**
+ * Checks if items in Grouped Product are in stock
+ * @param product: productGroup
+ * @param products: products in stock
+ * @namespace Util/Product/Extract/getGroupedProductsInStockQuantity */
+
+export const getGroupedProductsInStockQuantity = (product) => {
+    const { items = [] } = product;
```

Comment:
You can destruct it in argument then it will be possible to make function shorter.

Reply:
Done